privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -1216,6 +1217,31 @@ def check_otp(self, otpval, counter=None, window=None, options=None):
             user_handle = getParam(options, "userhandle", optional)
             assertion_client_extensions = getParam(options, "assertionclientextensions", optional)
 
+            # Check if a whitelist for AAGUIDs exists, and if this device is whitelisted. If not raise a
+            # policy exception.
+            allowed_aaguids = getParam(options, WEBAUTHNACTION.AUTHENTICATOR_SELECTION_LIST, optional)
```

Comment:
i dont think we should create a dependency on `Match` for the tokenclasses if they are the lowest layer in our design

Reply:
But we already have a few token classes which use `Match`... But yeah, this is probably a redesign issue :-)

Reply:
i guess the idea is not bad, since the token class itself can specify policies. However i would prefer a more uniform abstraction (aka an object), not just dicts (which are currently also used for specifying the policies)

Reply:
what i mean to say is, we should deal with both at the same time